packages/graphql-yoga/__tests__/node.spec.ts

```diff
@@ -1481,3 +1482,58 @@ describe('404 Handling', () => {
     expect(body).toEqual('Do you really like em?')
   })
 })
+
+describe('Respect Accept headers', () => {
+  const yoga = createYoga({
+    schema,
+  })
+  const server = createServer(yoga)
+  let port: number
+  let url: string
+  beforeAll((done) => {
+    port = Math.floor(Math.random() * 100) + 4000
+    url = `http://localhost:${port}/graphql`
+    server.listen(port, done)
+  })
+  afterAll(() => {
+    server.close()
+  })
```

Comment:
I think these tests do not necessarily require running

Reply:
I think it is safer to test stream requests and responses via the really HTTP